### ISSUE TITLE
Typo and off-by-one error in stratifying/jittering section of Book 3

### ISSUE THREAD

**Issue by TheRayTracer** [RayTracing/raytracing.github.io#976]
I have noticed a small typo in the stratifying section of Book 3 of the dev-major v4 branch. The typo is in listing 6.

`int sqrt_spp = int(sqrt(samples_per_pixrel));`

Also, in listing 6, I believe there is an off-by-one error. This has existed since v3 of Book 3 and the accompanying reference source code.

```
auto u = (i + (s_i + random_double()) / sqrt_spp) / (image_width-1);
auto v = (j + (s_j + random_double()) / sqrt_spp) / (image_height-1);

```
Should be:
```
auto u = (i + (s_i + random_double()) / sqrt_spp) / (image_width);
auto v = (j + (s_j + random_double()) / sqrt_spp) / (image_height);

```

And also in the final scene.h source code.

**Comment by trevordblack** [comment]
Thanks. I'll look into it.

**Comment by trevordblack** [comment]
This is a sibling issue to #682

**Comment by hollasch** [comment]
Yes, this is being addressed in issue #682.

**Comment by hollasch** [comment]
Fixed the typo.

**Comment by trevordblack** [comment]
`samples_per_pixrel` still needs to be addressed

**Comment by hollasch** [comment]
I've fixed it in the feature branch (not yet on GitHub).

**Comment by hollasch** [comment]
Ref #993

**Comment by hollasch** [comment]
Two issues here. One is fixed, and the other is addressed in #682. Closing.
